Harvest.Web/Controllers/Api/QuoteController.cs

```diff
@@ -53,8 +53,13 @@ public async Task<ActionResult> GetApproved(int projectId)
             var model = new QuoteModel
             {
                 Project = project,
-                Quote = !string.IsNullOrWhiteSpace(project.Quote?.Text) ? QuoteDetail.Deserialize(project.Quote.Text) : null
+                Quote = !string.IsNullOrWhiteSpace(project.Quote?.Text) ? QuoteDetail.Deserialize(project.Quote.Text) : null,
             };
+            if (model.Quote != null)
+            {
+                model.Quote.ApprovedBy = project.Quote?.ApprovedBy;
+                model.Quote.ApprovedOn = project.Quote?.ApprovedOn;
+            }
```

Comment:
This is fine but you can also check `project.Quote != null` in the if statement and your code below would be cleaner.  obviously pretty minor stuff.